### ISSUE TITLE
Registering formats problem

### ISSUE THREAD

**Issue by simon-p-r** [zaggino/z-schema#138]
Hi

I have encountered a problem after registering formats in my unit tests where the state persists to my next test.  Not sure how to fix it at the moment, I have tried adding a static method on zSchema object to unregister after each test but this is very messy.

Thanks
Simon

**Comment by zaggino** [comment]
Try
```
var FormatValidators  = require("z-schema/src/FormatValidators");
delete FormatValidators[formatName];
```

**Comment by zaggino** [comment]
`ZSchema.unregisterFormat("xxx");` available in `3.15.1`

**Comment by simon-p-r** [comment]
Thanks :+1:
